### DISCUSSION TITLE
Customize config folder path

### DISCUSSION THREAD

**Discussion by Iameora** [node-config/node-config#718] [Q&A] [answered]
Hello,
I'm not been able to find the way to change default config folder path, how I can do it?
I have multiple local app (in a different folders) and wish to make them to share same config file since info required are the same
Thanks in advance

**Comment by markstos** [comment, marked as answer]
Have you tried setting NODE_CONFIG_DIR? 

https://github.com/node-config/node-config/wiki/Environment-Variables#node_config_dir
